### PR TITLE
Feature/icon on hover

### PR DESCRIPTION
**Pull request by eileenabadi**
icon shows only on hover

### CODE REVIEW

File: client/src/app/main/machine/list/machine-list.component.html [hunk 1]
```diff
@@ -4,38 +4,37 @@
       <table class="table">
         <thead>
           <tr>
-            <th width="20%" (click)="tableSortService.sortTable(sortProperties.AssetName, machines)">Asset Name</th>
-            <th width="20%" (click)="tableSortService.sortTable(sortProperties.ApplicationEnvironment, machines)">
+            <th width="20%" (click)="machines ? tableSortService.sortTable(sortProperties.AssetName, machines) : null">Asset Name</th>
+            <th width="20%" (click)="machines ? tableSortService.sortTable(sortProperties.ApplicationEnvironment, machines) : null">
```
Comment (mischaboldy) [inline client/src/app/main/machine/list/machine-list.component.html]: wouldn't it be easier to check if machines is undefined in the sortservice?
Also, I assume that even though the table is empty you can still hover over the tabs and they have a hover effect, even though nothing can technically happen. Not sure if that is wanted behavior

File: client/src/styles/_tables.scss [hunk 2]
```diff
@@ -116,3 +120,11 @@ $tableHeaderBorder: solid $themeColorOne 1px;
...
   }
 }
+
+table.table fa.table-icon {
+  visibility: hidden;
+}
+
+table.table tr:hover fa.table-icon{
+  visibility: visible;
+}
```
Comment (mischaboldy) [inline client/src/styles/_tables.scss]: missing a space after `table-icon`

Also does this make them all visible? or just the row you are hovering over. I'm not sure.

Comment (eileenabadi) [reply client/src/styles/_tables.scss]: it's not all visible, only for that specific row

File: client/src/app/main/machine/list/machine-list.component.html (hunk 1 above)
Comment (eileenabadi) [reply client/src/app/main/machine/list/machine-list.component.html]: i haven't found a way to set hover on condition. i'll look further monday

Comment (mischaboldy) [reply client/src/app/main/machine/list/machine-list.component.html]: No need to set hover on condition. you can give it a conditional class, something like
`[class.hoverable]="machines"`

File: client/src/styles/_tables.scss (hunk 2 above)
Comment (mischaboldy) [reply client/src/styles/_tables.scss]: then ignore my comment :)

**Pull request merged**